source/Nevermore/ICompleteQuery.cs

```diff
@@ -18,11 +18,13 @@ public interface ICompleteQuery<TRecord> where TRecord : class
         /// <returns>Returns true if there are any rows in the result set, otherwise false</returns>
         bool Any();
 
+        [Obsolete("Use FirstOrDefault instead")]
```

Comment:
`Use FirstOrDefault instead` might not be enough for someone to understand why then need to make the change..  

Should we extend the explanation to something like this:
`First returns the first row, or null if there are no rows To make your code easier to read use FirstOrDefault  instead.`

Reply:
Instead of making this `Obsolete`, should we just remove this instead? Saves having two methods around for a while, and saves having to come back and remove this later (if we decide to). I don't _think_ there are many people other than us that use Nevermore. When we have made breaking changes in the past, I don't think anyone has complained.

Reply:
Good feedback, thanks :+1: I've thought a bit about this and I think the benefit from the explanation/hint for upgrading users (rather than just a broken build) outweighs the cost of having two methods around for a while (which seems pretty harmless, given nice IDE treatment of `Obsolete` methods) and the cost of having to come back and remove this later (which is optional, and trivial).

I agree we might be fine to make this is a breaking change, but it's just _so easy_ to make it non-breaking in this case :-)